src/terminal/adapter/DispatchTypes.hpp

```diff
@@ -212,7 +239,9 @@ namespace Microsoft::Console::VirtualTerminal
         static constexpr VTParameter defaultParameter;
         static constexpr std::span defaultParameters{ &defaultParameter, 1 };
 
-        std::span<const VTParameter> _values;
+        std::span<const VTParameter> _params;
+        std::span<const VTParameter> _subParams;
+        std::span<const std::pair<BYTE, BYTE>> _subParamsRange;

```

Comment:
the like, smallest nit ever: I might rename this to `_subParamRanges`. For some reason, it makes more sense in my head if you flip which nolun is plural. Somhow that makes it more clear that it's a set of ranges that contains the subparams for each param. 

Feel free to ignore me though :relaxed:

Reply:
Yeah, it makes sense.. I'll update the PR. Thanks.

Reply:
made the requested changes👍